src/foam/u2/detail/SectionView.js

```diff
@@ -78,6 +78,24 @@ foam.CLASS({
           return this.getNestedPropValue(obj[props.shift()], props.join('.'))
         }
       }
+    },
+    {
+      class: 'Boolean',
+      name: 'loadLatch',
+      expression: function(selected) {
```

Comment:
You don't need this expression and preSet and the postSet on 'selected'. Either approach will work. The postSet is more efficient.

Reply:
I think I need expression for such use case:
if `selected` was not changed then this expression will set `loadLatch` to true as `selected` is true otherwise loadLatch will be false, which we don't want in cases on SectionView being used elsewhere other then TabbedDetailView

Reply:
Ok, then you should document that, since it isn't going to be obvious to the casual observer, who might be tempted to "fix" your code.